gitlab-kas.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: gitlab-kas
-  version: 14.5.0
+  version: 16.4.0
```

Comment:
This package has `update` disabled ... is it possible to enable update for this, or add a comment to explain why we can't?

Reply:
Let's enable it. Question: Which value do we set to the `release-monitor` ?

Reply:
Okay, I figured it out. I couldn't find a release-monitor for this project. What should we do under these situations ?